pkg/dataconn/wire.go

```diff
@@ -6,6 +6,11 @@ import (
 	"fmt"
 	"io"
 	"net"
+	"unsafe"
+)
+
+const (
+	requestHeaderSize = 26
```

Comment:
I prefer to make this as a variable in `Wire` and it would be calculated during the init of the struct.

Reply:
Could you make this `unsafe.Sizeof(msg.MagicVersion) + unsafe.Sizeof(msg.Seq) ...`?

Reply:
Sure.

Reply:
Updated.